enforcer/utils/ksmutil.c

```diff
@@ -1669,6 +1670,7 @@ cmd_exportkeys ()
         hsm_print_error(NULL);
         exit(-1);
     }
+    ctx = hsm_create_context();
```

Comment:
Our hsm could very well fail to clone context. We should check for NULL. Since this is just ksmutil we could just bail out.